secator/celery_signals.py

```
@@ -15,6 +15,31 @@
+def clear_shutdown_flag():
+	"""Remove the eviction flag (called on worker boot to drop any stale flag)."""
+	if SHUTDOWN_FLAG.exists():
+		SHUTDOWN_FLAG.unlink()
+
+
+def worker_shutting_down_handler(**kwargs):
+	"""Raise the eviction flag so the in-flight task's monitor stops it early and returns."""
+	try:
+		SHUTDOWN_FLAG.write_text("1")
+	except Exception:
+		pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)